core/spec/models/spree/product_spec.rb

```diff
@@ -222,11 +222,10 @@ def duplicate_extra(old_product)
         end
 
         it "cannot create another product with the same permalink" do
-          pending '[Spree build] Failing spec'
           @product2 = create(:product, :name => 'foo')
           lambda do
             @product2.update_attributes(:permalink => @product1.permalink)
-          end.should raise_error(ActiveRecord::RecordNotUnique)
+          end.should raise_error(ActiveRecord::StatementInvalid, /SQLite3::ConstraintException/)
```

Comment:
Is there a more general error we can test here? We actually don't want to use sqlite.

Reply:
I was able to test with Postgres and it doesn't fail ! Should we keep `AR::RecordNotUnique` then ?

Reply:
Yes, the fewer patches we need, the better. :-) Thank you for testing.